dll/win32/shell32/wine/shellord.c

```diff
@@ -1388,8 +1388,15 @@ HRESULT WINAPI SHWinHelp(HWND hwnd, LPCWSTR pszHelp, UINT uCommand, ULONG_PTR dw
  */
 BOOL WINAPI SHRunControlPanel (LPCWSTR commandLine, HWND parent)
 {
+#ifdef __REACTOS__
+    HINSTANCE hr;
+    TRACE("SHRunControlPanel(%s, %p)n", debugstr_w(commandLine), parent);
+    hr = ShellExecuteW(parent, NULL, commandLine, NULL, NULL, SW_NORMAL);
```

Comment:
'If the specified Control Panel item is already running, SHRunControlPanel attempts to switch to that instance rather than opening a new instance.'
Does `ShellExecuteW` handle that?

Reply:
We already have tickets on this : 
https://jira.reactos.org/browse/CORE-7921
https://jira.reactos.org/browse/CORE-17025

Using this central function may help to manage it more easily.

Reply:
I'm adding your comment in the "TODO" section

Reply:
> We already have tickets on this

Right.
Maybe add their IDs to the comment?